gulp-core-build/typings/jju/jju.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for jju 1.3.0
```

Comment:
Can we name it typings-custom instead (to differentiate from typings/tsd) and use typesRoot to pull this in, per TS 2.0 conventions?